src/mini_trainer/train.py

```
@@ -221,7 +234,125 @@ def save_model(
 
     log_rank_0("")
     torch.distributed.barrier()
-    log_rank_0(f"✅ Saved model at {samples_seen} samples in {time.time() - start:.2f} seconds")
+    log_rank_0(f"✅ Saved model at {samples_seen} samples{suffix_text} in {time.time() - start:.2f} seconds")
+
+def compute_validation_loss(model, val_data_loader, device, world_size):
```

Ruff (0.12.2)

Unused function argument: `world_size`

(ARG001)